core/src/main/java/org/infinispan/manager/impl/ClusterExecutorImpl.java

```diff
@@ -72,7 +72,6 @@ public ClusterExecutorImpl(Predicate<? super Address> predicate, EmbeddedCacheMa
 
    /**
     * Returns the targets we should use for JGroups.  This excludes the local node if it is a target.
-    * @return
```

Comment:
I'm not a big fan of duplicated text. And IntelliJ flags tags with missing text as warnings :)